src/pages/Landing/Landing.css

```diff
@@ -8,12 +8,21 @@
   background-size: cover;
   /* background-position: center; */
 
+  @media screen and (prefers-color-scheme: dark) {
+    background-blend-mode: multiply;
+    background-color: DimGray;
```

Comment:
This is a trick borrowed from Photoshop's Layer Blending to darken the background image.